### PR TITLE
Enable mount namespacing for container mounts in gcp-rt lane

### PR DESCRIPTION
**Pull request by MarSik**

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml [hunk 1]
```diff
@@ -325,6 +325,8 @@ tests:
     cluster_profile: gcp
     env:
       RT_ENABLED: "true"
+    pre:
+    - ref: telco-isolate-mount
```
Comment (petr-muller) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml:329]:
```suggestion
    - chain: ipi-conf-gcp
    - ref: telco-isolate-mount
    - chain: ipi-install
```

**Review by petr-muller — COMMENTED** [review]
One change needed - providing custom `pre:` is a full override, so you also need to provide the "rest" of the original workflow.

**Comment by yuvalk** [comment]
/close
closing in favor of #23952

**Pull request closed without merging**